tools/autograd/derivatives.yaml

```diff
@@ -540,6 +540,10 @@
 - name: i0(Tensor self) -> Tensor
   self: not_implemented("i0")
 
+- name: igamma(Tensor self, Tensor other) -> Tensor
+  self: 'not_implemented("igamma: a")'
```

Comment:
`a` is the first argument of incomplete gamma, usually written as `igamma(a, x)`.
If you think it causes more confusion, I can change it to `self`.

Reply:
The doc calls it "input" and the signature "self". Maybe renaming them all to the same thing will make this clearer indeed.

Reply:
Our idiosyncratic convention is that signature is self and docs are input. It should be referred to using the name in the docs.

Reply:
Removing the single quotes makes the build failed. I guess because there is a space in the string. But I've changed it to `igamma: self` in the latest commit

Reply:
I think it's "igamma: input" here to be consistent with the docs?

Reply:
Ah, yes, sorry, a bit confused. Fixed this.